cypress/support/index.js

```diff
@@ -18,5 +18,11 @@ import './commands';
 import 'cypress-axe';
 import '@testing-library/cypress/add-commands';
 
+// Delete window.fetch on every window load so we can
+// intercept tests using whatwg-fetch
+Cypress.on('window:before:load', win => {
+  delete win.fetch;
+});
+
```

Comment:
We need to polyfill the `react-geocode` libs `fetch` request in order to intercept/assert the request string. This is probably a good idea anyways to support older browsers. This tells cypress to use that polyfill instead of it's own implementation registry.

Reply:
Off of @racedale's comment, I think that this is where the Netlify build is failing. We may need to check to see if `window` is defined first, before deleting fetch.